src/native/_auth.js

```diff
@@ -277,7 +277,7 @@ const makeShareMDataPermissions = (permissions) => {
   return new ShareMDataArray(permissions.map((perm) => {
     const permsArray = helpers.makePermissionSet(perm.perms);
     return ShareMData({
-      type_tag: perm.type_tag,
+      typeTag: perm.typeTag,
```

Comment:
I think here we also need to keep `typeTag: perm.typeTag,` as I think both structs (in and out) are used by ffi, i.e. they are internal not a json we expose.